### PR TITLE
feat: Include workflow to create draft release

### PR DESCRIPTION
**Pull request by CristhianMotoche**
Solves issue #74

# TODO:
- [x] Provide a Github token
- [x] Test behavior

---

As part of [Stack Builders Inc.](https://www.stackbuilders.com) Hacktoberfest 2023 event.

### CODE REVIEW

**Comment by CristhianMotoche** [comment]
It seems this is already working! Here is an example: https://github.com/stackbuilders/inflections-hs/releases

File: .github/workflows/draft.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: Release package to Github
+
+on:
+  push:
+    branches:
+      - fix/solve-issue-74
+    # tags:
+    #   - 'v*'
```
Comment (Centeno448) [inline .github/workflows/draft.yml:8]: Should probably remove these comments, or do they serve a documentation purpose?

File: .github/workflows/draft.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      - name: Create release with package assets
+        uses: softprops/action-gh-release@v1
+        with:
+          body_path: CHANGELOG.md
+          files: |
+            speed-benchmark.txt
+            memory-benchmark.txt
+          draft: true
+        env:
+          # TODO: Get a GITHUB_TOKEN
```
Comment (Centeno448) [inline .github/workflows/draft.yml:57]: Would I need to follow up on this token generation on my side?

**Review by Centeno448 — COMMENTED** [review]
Left a few questions 😄

File: .github/workflows/draft.yml (hunk 1 above)
Comment (CristhianMotoche) [reply .github/workflows/draft.yml:8]: Good catch! I need to remove those.

**Comment by CristhianMotoche** [comment]
Left some responses :grin:

**Review by Centeno448 — APPROVED** [review]
Sweet, thanks @CristhianMotoche

**Pull request merged**